moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -227,22 +185,56 @@ const ProjectsListViewTable = () => {
         />
         {/*Main Table Body*/}
         <Paper className={classes.paper}>
-          <Box mt={3}>
-            {!data && !error ? (
-              <CircularProgress />
-            ) : data && data.project_list_view ? (
-              <Card className={classes.root}>
-                <MaterialTable
-                  onChangeColumnHidden={handleColumnChange}
-                  data={data.project_list_view}
-                  columns={columns}
-                  title=""
-                  options={tableOptions}
-                  components={tableComponents}
-                />
-              </Card>
-            ) : (
-              <span>{error ? error : "Could not fetch data"}</span>
+          <Box sx={{ height: "600px", padding: "14px", marginTop: "24px" }}>
```

Comment:
ahhh thank you for the screenshot, i was wondering what it would look like on a different / bigger monitor than the ones ive been using, ill try with the vh!

Reply:
this is a great catch, Frank! I agree that vh would be a good route. I faced this same problem with the [projects map layout](https://github.com/cityofaustin/atd-moped/pull/1276), and it made me wonder if we should spin off an issue to make this uniform across our main views like dashboard, projects, project page, and staff. idk maybe some of them needed different treatment. 🤷

Reply:
I changed it to be dynamic, thank you for that recommendation. Let me know if you think it can refined more

Reply:
oops my comment is outdated now. The new code looks ✨